docs/basics/primitives.mdx

```diff
@@ -3,8 +3,8 @@ title: Primitives
 nav: 1.02
 ---
 
-State in Jotai is a set of atoms.
-An atom is a piece of state.
+An atom in jotai represents a piece of state. It can hold any value ranging from string, boolean or number to complex structures like arrays, objects, functions and so on. You can consider them the building blocks of your app state.
```

Comment:
A gotcha is that we cant store a function directly in an atom. We need to wrap it with an object.

Reply:
@dai-shi Yeah I was thinking about it too. But I still feel like it's important to mention it, even if it may be a bit confusing at first.

The key is that I have to add somewhere that you can store anything, maybe mentioning more examples like Blob or Base64 or you name it... BUT, in the case of a function, you just need to store it inside an objet.
Let me find the good spot

Reply:
@dai-shi Last commit contains my proposition

Reply:
@TwistedMinda  Also if you don't mind, could you capitalize Jotai for consistency with the rest of the docs. 🙂

Reply:
Added my new proposition :)